sample/src/androidTest/java/com/schibsted/spain/barista/sample/RecyclerViewInsideViewPagerTest.java

```diff
@@ -0,0 +1,40 @@
+package com.schibsted.spain.barista.sample;
+
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static com.schibsted.spain.barista.BaristaAssertions.assertDisplayed;
+import static com.schibsted.spain.barista.BaristaRecyclerViewActions.clickRecyclerViewItem;
+import static com.schibsted.spain.barista.BaristaRecyclerViewActions.clickRecyclerViewItemChild;
+import static com.schibsted.spain.barista.BaristaRecyclerViewActions.scrollTo;
+
+@RunWith(AndroidJUnit4.class) public class RecyclerViewInsideViewPagerTest {
```

Comment:
Do we have a guideline for formatting of class annotations? I think this is the only place where the class name and the runner are in the same line.